SparkPost/2web2ui
# SA-1819: Sent correct query parameters to report tabs

### What Changed
 - Comparators are now properly sent to report tabs

### How To Test
 - Add new filters
 - Add metrics that show each tab
   - Bounce metrics
   - Engagement (like count)
  - Delays
   - Rejections
 - Verify that the filters are properly sent to the request

### To Do
